### PR TITLE
Add Sign Up button to Log In page

### PR DESCRIPTION
**Pull request by neoxelox**
Added Sign Up button to Log In Page.

### CODE REVIEW

File: apps/web/src/components/layouts/FocusLayout.tsx [hunk 1]
```diff
@@ -2,16 +2,19 @@ import { ReactNode } from 'react'
 
 export default function FocusLayout({
   header,
+  footer,
   children,
 }: {
   header: ReactNode
+  footer: ReactNode
   children?: ReactNode
 }) {
```
Comment (csansoon) [inline apps/web/src/components/layouts/FocusLayout.tsx:11]: `FocusLayout` is being used in more places, we should make `footer` optional.

File: packages/web-ui/src/ds/molecules/FocusFooter/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { ReactNode } from 'react'
+
+export default function FocusFooter({ content }: { content: ReactNode }) {
```
Comment (andresgutgon) [inline packages/web-ui/src/ds/molecules/FocusFooter/index.tsx:3]: This is super specific for the signup. Doesn't look like something part of design system.

File: apps/web/src/components/layouts/FocusLayout.tsx [hunk 3]
```diff
@@ -2,16 +2,19 @@ import { ReactNode } from 'react'
...
   return (
     <div className='flex flex-col items-center justify-center h-screen'>
       <div className='flex flex-col gap-y-6 max-w-[22rem]'>
         <div>{header}</div>
         {children}
+        <div>{footer}</div>
```
Comment (andresgutgon) [inline apps/web/src/components/layouts/FocusLayout.tsx:17]: Do an if here and put that focus footer div styles
```javascript
{footer ? (<div classNmame="...">{footer}</div>) : null}
```

**Review by andresgutgon — APPROVED** [review]

File: apps/web/src/components/layouts/FocusLayout.tsx (hunk 3 above)
Comment (csansoon) [reply apps/web/src/components/layouts/FocusLayout.tsx:17]: Why the styling there? The header is not doing that

Comment (andresgutgon) [reply apps/web/src/components/layouts/FocusLayout.tsx]: Good point. Because people change opinions. I did the header and now I changed my opinion.

**Pull request merged**